src/pages/Schedule/index.jsx

```diff
@@ -0,0 +1,239 @@
+import { useEffect, useState } from 'react';
+import { Helmet } from "react-helmet";
+import { useLocalStorage } from "usehooks-ts";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import Fab from '@mui/material/Fab';
+import AddIcon from '@mui/icons-material/Add';
+import DeleteIcon from "@mui/icons-material/Delete";
+import LockIcon from "@mui/icons-material/Lock";
+import LockOpenIcon from "@mui/icons-material/LockOpen";
+import Paper from '@mui/material/Paper';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import TableCell from '@mui/material/TableCell';
+import TableBody from '@mui/material/TableBody';
+import Table from '@mui/material/Table';
+import TextField from '@mui/material/TextField';
+import Select from '@mui/material/Select';
+import MenuItem from '@mui/material/MenuItem';
+import Checkbox from '@mui/material/Checkbox';
+
+export default function Schedule() {
+  const keyOptions =
+    [
+      "--ceph",
+      "--ceph-repo",
+      "--suite-repo",
+      "--suite-branch",
+      "--suite",
+      "--subset",
+      "--machine",
+      "--filter",
+      "--distro",
+      "--rerun",
+      "--rerun-statuses",
+      "--limit",
+      "--priority",
+      "--force-priority"
+    ];
+
+  const [rowData, setRowData] = useLocalStorage("rowData", []);
+  const [rowIndex, setRowIndex] = useLocalStorage("rowIndex", -1);
+  const [commandBarValue, setCommandBarValue] = useState([]);
+
+  useEffect(() => {
+    setCommandBarValue(rowData);
+  }, [rowData])
+
+  const handleRun = () => {
+    return false;
+  };
+
+  const handleDryRun = () => {
+    return false;
+  };
+
+  const addNewRow = () => {
+    console.log("addNewRow");
+    const updatedRowIndex = rowIndex + 1;
+    setRowIndex(updatedRowIndex);
+    const index = (updatedRowIndex % keyOptions.length);
+    const object = {
+      key: keyOptions[index],
+      value: "",
+      lock: false,
+      checked: true,
+    }
+    const updatedRowData = [...rowData];
+    updatedRowData.push(object);
+    setRowData(updatedRowData);
+  };
+
+  const handleCheckboxChange = (index, event) => {
+    console.log("handleCheckboxChange");
+    const newRowData = [...rowData];
+    if (event.target.checked) {
+      newRowData[index].checked = true;
```

Comment:
I think it might be good to add a condition that "value" of the row is not empty when checking the box so we don't have empty `--ceph` in the command bar without a value.

Reply:
Please ignore other outdated comments, looks like I had some old "pending" comments which never got submitted.

Reply:
> I think it might be good to add a condition that "value" of the row is not empty when checking the box so we don't have empty `--ceph` in the command bar without a value.

Agree, but need to watch out for flags like --force-priority, honestly I think we can just make --force priority a default.

Reply:
From what I understand, `--force-priority` schedules tests immediately without checking the priority. So we'll probably not want that as default. According to the documentation `--force-priority` is to be only used in emergencies: https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/tests-integration-testing-teuthology-intro/#testing-priority

We can probably have default values for each field, example `--ceph` can have `main`. And `--force-priority` can have `0` or `False` as default so people would get an idea for how to enable it. What do you think?

Reply:
Just got back to this, we make force priority a separate button, so this conversation has been resolved.